src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipProcessor.kt

```diff
@@ -14,7 +14,8 @@ import exchange.dydx.abacus.utils.NATIVE_TOKEN_DEFAULT_ADDRESS
 import exchange.dydx.abacus.utils.mutable
 import exchange.dydx.abacus.utils.safeSet
 
-@Suppress("NotImplementedDeclaration", "ForbiddenComment")
+private const val UNISWAP_SUFFIX = "uniswap"
```

Comment:
for my understanding is uniswap the only dex supported?

Reply:
yup! right now we're just hardcoding it but i'm considering moving to statsig 🤔 so we can be more nimble.